### DISCUSSION TITLE
Combine several RTSP streams in one HLS playlist

### DISCUSSION THREAD

**Discussion by jvelo** [bluenviron/mediamtx#2343] [Questions]
Hello,

I have cameras that expose different 3 RTSP streams : one per resolution. Is it be possible using mediamtx to expose one HLS playlist that would expose the different resolutions so that the client can support adaptive bitrate ?

Thanks in advance

**Comment by jvelo** [comment]
As I understand, there's always the possibility to define the multi-variant playlist outside of mediamtx – still interested in knowing if that can be done through configuration directly in mediamtx.

**Comment by aler9** [comment]
Hello, this feature is currently missing, to track the development follow this enhancement request: #1948

[a comment by an automated account is not shown]
